### PR TITLE
Refactor(BCE-214): Add Responsiveness To Manage Page

### PR DESCRIPTION
**Pull request by Funnyadd**
JIRA: [BCE-214](https://champlainsaintlambert.atlassian.net/browse/BCE-214)
## Context:
Make UI front end of manage page responsive and available in french and english

## Changes
- Made the client grading table responsive
- Added translation for the client grading table
- Reformatted the errors for the client grading form to make it work with the new architecture
- Made the dashboard fully responsive to language change
- Fixed issue where the language selector would show two times the languages
- Added images to the languages in the language selector
- Fixed the issue where the language dropdown would overflow when clicked on

#### Before
managebefore
managerezizebefore

#### After
manageafter
manageresizeafter

#### Before
languagebefore

#### After
languageafter

#### Before
dashboardbefore

#### After
dashboardafter

## Linked Branch
[Feat/BCE-28/Modify_Client_Grading_Brackets](https://github.com/BC-Engine-Team/BC-Engine/tree/feat/BCE-28/Modify_Client_Grading_Brackets)

### CODE REVIEW

**Review by cjayneb — APPROVED** [review]
Nice. Now we can use Client grading on ipad. I didnt check the backend methods since they will be modified and this PR is purely for the frontend. LGTM.

**Review by NickEspitalier — APPROVED** [review]
Looks great and thank you so much for fixing the UI responsiveness, however I have only one question.

File: client/src/locales/fr/translation.json [hunk 1]
```diff
@@ -121,7 +121,28 @@
...
+        },
+         "error" : {
+            "Higher": "Doit être plus grand que les notes plus élevées.",
+            "Lower": "Doit être plus petit que les note plus basses.",
+            "Min": "Le minimum doit être plus petit que le maximum.",
+            "Dropdown": "Veuillez selectionner un option.",
+            "NoNum": "Veuillez entrer un chiffre."
+         }
+       },
+       "employeeTable": {
```
[the remark this answers is not in the document]
Comment (Funnyadd) [reply client/src/locales/fr/translation.json:145]: no problem borther

File: client/src/components/NavB.js [hunk 2]
```diff
@@ -25,7 +29,7 @@ const NavB = (props) => {
     const GreetingLabel = t('navbar.Greeting');
     const SignOutLabel = t('navbar.SignOutLabel');
 
-    const [languageTitle, setLanguageTitle] = useState(lngs[i18n.language].nativeName);
+    const [languageTitle, setLanguageTitle] = useState(lngs['en'].nativeName);
```
[the remark this answers is not in the document]
Comment (Funnyadd) [reply client/src/components/NavB.js]: doesn't mather, it's just to initialize the varaible, later, in a useState, it changes it before the page is rendered

[the remark this answers is not in the document]
Comment (NickEspitalier) [reply client/src/components/NavB.js]: Ok makes sense 👍

**Pull request merged**